src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const personPartnerType = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age && <p className="Person__age">I am {person.age}</p>}
+
+      <p className="Person__partner">
+        {person.isMarried
+          ? `${person.partnerName} is my ${personPartnerType}`
+          : 'I am not married'}
+      </p>
```

Comment:
Still not fixed, move this logic to the helper variable and use it here 
<img width="813" alt="image" src="https://github.com/user-attachments/assets/4c19ac11-28c1-4ae1-92db-61a8c56ea585">